roles/matomo/README.adoc

```diff
@@ -0,0 +1,62 @@
+
+== Install it
+
+Prior to installing Matomo with IIAB, the default URL (http://box.lan/matomo) can be customized in https://wiki.iiab.io/go/FAQ#What_is_local_vars.yml_and_how_do_I_customize_it.3F[/etc/iiab/local_vars.yml]
+
+One way to do that is by changing these 2 lines:
+
+----
+iiab_hostname: box
+iiab_domain: lan
+----
+
+Or, you can change the Matomo URL by putting your IIAB IP Address in a line like:
+
+----
+matomo_host_url: http://192.168.0.199
+----
+
+Either way, consider setting a Matomo username and password using lines like:
+
+----
+matomo_db_user: Admin
+matomo_db_pass: changeme
+----
+
+Also ensure that your `/etc/iiab/local_vars.yml` contains these lines:
+
+----
+matomo_install: True
+matomo_enabled: True
+----
+
+_Finally, continue to https://download.iiab.io[install IIAB], e.g. by running `sudo iiab`, until software installation is complete._
+
+== Use it
+
+Log in to your IIAB's full Matomo URL, e.g. http://box.lan/matomo, as arranged above.
+
+Take a look at Matomo's official guides to further set this up: https://matomo.org/guides/
+
+WARNING: Matomo won't show any traffic statistics until after 1 day or reboot (which are the events that trigger the log scraper!)
+
+=== Getting Started
+
+Matomo is developed with commercial websites in mind. After navigating to http://box.lan/matomo and logging in with the user name and password you set above, you will see a variety of references to revenue, marketplaces, and other terms focused on commercialization and advertising. Don't worry about that.
+
+The heart of Matomo's value for you is in the navigation bar on the left side of the page. Click on "Visitors" and then below "Visitors", "Overview", to see how many different users are visiting your site. The top of the page will show a graph of how many visits occur on each day (although your device can't keep track of time when it is off and has no connection to the Internet, so this graph might not be perfectly accurate). Below the graph, you'll see some overall statistics, like how many unique visitors you've had. Matomo thinks of visitors in terms of devices, so it won't know if two people are connecting to Internet in a Box using the same phone. There are several other interesting statistics here, like the average visit duration, or average time your visitors are spending using Internet in a Box.
+
+Below the "Visitors" button is a second button, "Behavior". Click on the "Pages" button after clicking "Behavior" and you can see the various pages that have been visited by your users. You may not see activity from the most recent day, since Matomo only updates its records once per day.
+
+=== IIAB Tips, Tricks, and Gotchas
+
+1. If your Internet In A Box setup is without power and Internet access, it will not be a ble to keep time correctly. This is okay! But it means that the time-of-visit information in Matomo will not be correct.
```

Comment:
```suggestion
1. If your Internet-in-a-Box setup is without power and Internet access, it will not be able to keep time correctly. This is okay! But it means that the time-of-visit information in Matomo will not be correct.
```